### ISSUE TITLE
False positives with `no-empty-files`

### ISSUE THREAD

**Issue by andreavaccari** [sindresorhus/eslint-plugin-unicorn#1598]
The rule `no-empty-files` triggers false positives for [`.d.ts` modules](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) that contain only [triple-slash directives](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).

This is used, for example, by vite in lieu of `compilerOptions.types` in `jsconfig.json` or `tsconfig.json` ([ref](https://github.com/vitejs/vite/tree/main/packages/create-vite/template-svelte-ts#technical-considerations))

Example:

```js
/// <reference types="svelte" />
/// <reference types="vite/client" />
```

**Comment by manovotny** [comment]
Awesome catch! I'll check it out.

**Comment by manovotny** [comment]
@fisker PR submitted (#1605). Let me know what you think!
